pkg/flow/internal/controller/node_config_import.go

```diff
@@ -236,6 +241,21 @@ func (cn *ImportConfigNode) onContentUpdate(importedContent string) {
 	cn.OnBlockNodeUpdate(cn)
 }
 
+func compareMaps(map1, map2 map[string]string) bool {
+	if len(map1) != len(map2) {
+		return false
+	}
+
+	for key, value1 := range map1 {
+		value2, ok := map2[key]
+		if !ok || value1 != value2 {
+			return false
+		}
+	}
+
+	return true
+}
```

Comment:
nit: can we use [maps.Equal](https://pkg.go.dev/maps#Equal) from the standard library instead?